test/unit/scanner-engine.test.ts

```diff
@@ -22,10 +22,10 @@ import axios from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import { ChildProcess, spawn } from 'child_process';
 import fs from 'fs';
```

Comment:
sorry, I missed that before. Let's make sure we only use fs-extra everywhere instead of native fs.

Reply:
done